tests/DependencyInjection/RabbitMqConsumerHandlerExtensionTest.php

```diff
@@ -117,33 +87,62 @@ public function testConfigureContainerParameter(
 		$this->compile();
 	}
 
-	public function testConfigureCustomLoggerInstance(): void
+	/**
+	 * @return mixed[][]|\Generator
+	 */
+	public function configureContainerServiceDataProvider(): Generator
```

Comment:
```suggestion
	public function configureContainerServiceAliasDataProvider(): Generator
```

This is not set up for "any" service configuration :)

Plus please the same should be reflected in commit message.

Reply:
True, fixed.